index.html

```diff
@@ -1,259 +1,313 @@
-
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="style.css" type="text/css">
-    <script src="https://kit.fontawesome.com/5d13e94c8b.js" crossorigin="anonymous"></script>
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-GFVB4JC5H5"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <script
+      src="https://kit.fontawesome.com/5d13e94c8b.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header class="header">
+        <div id="logo">
+            <a href="#"><h2 class="logo">Kayonga Chiteta</h2></a>
+        </div>
+      <nav class="d-none ml">
+        <i id="btn" class="d-none fas fa-bars two"></i>
+      </nav>
 
-  gtag('config', 'G-GFVB4JC5H5');
-</script>
-</head>
-<body>
-   <header class="header">
-       
-       <a href="#"><h2 class="logo">Kayonga Chiteta</h2></a>   
-       <nav class="d-none ml">
-           <i class=" d-none fas fa-bars two"></i> 
-    </nav>
-    <nav class="nav">
-        <ul class="nav-list d-none-m d-block">
-            <li>
-             Portfolio
-            </li>
-            <li>
-               About
-            </li>
-            <li>
-             Contact
-            </li>
-           <li> <img src="./photos/Union.png" alt="message icon"></li>
+      <nav class="nav  d-none-m">
+        <ul class="nav-list d-block">
+            <li class="close"><img id="close" src="./photos/Icon.png" alt=""></li>
+          <li class="common-li"><a href="#portfolio">Portfolio</a></li>
+          <li class="common-li"><a href="#about">About</a></li>
+          <li class="common-li"><a href="#contact">Contact</a></li>
+          <li class="common-li gone"><img src="./photos/Union.png" alt="#" /></li>
+          
         </ul>
-    </nav>
+      </nav>
```

Comment:
- [ ] Please kindly link the mobile menu options to the various sections of the page so that when a user clicks on a particular option, he/she should be taken to that section of the page. Check the screenshot of the requirements below and kindly fix that. 

![options](https://user-images.githubusercontent.com/62849872/150989299-0bb0f993-8de9-4069-b679-4577524d55ce.png)